frontend/src/pages/QueryBuilder/QueryBuilder.tsx

```diff
@@ -102,6 +102,11 @@ const QueryBuilder: FunctionComponent<QueryBuilderProps> = (props) => {
   const setActiveOperationByID = (id: string) => {
     const operation = props.operations.find((ope) => ope.get('id') === parseInt(id, 10));
     if (operation) {
+      if (operation.get('alias_creation_status') === 'p') {
```

Comment:
Check `!== 'd'` ... I'd rather flag this for error events as well.

Reply:
This is not very clear

Reply:
Very well ... replace this with:
`if (operation.get('alias_creation_status') !== 'd') {`

Reply:
Okay